Assets/GameAssets/World/WorldScripts/Boat/MetalBoatHealth.cs

```diff
@@ -0,0 +1,41 @@
+/*********************************************************************************************************************
+// File Name :         MetalBoatHealth.cs
+// Author :            Mark Hanson
+// Creation Date :     11/14/2024
+//
+// Brief Description : Manages the VFX of hitting the boat with the harpoon
+*********************************************************************************************************************/
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+/// <summary>
+/// Manages the VFX of hitting the boat with the harpoon
+/// </summary>
+public class MetalBoatHealth : BaseHealth
+{
+    /// <summary>
+    /// Overrides the base TakeDamage script to play the Wooden Spark VFX
+    /// </summary>
+    /// <param name="damage">amount of incoming damage</param>
+    /// <param name="damageSource">incoming damage source</param>
+    public override void TakeDamage(float damage, IBaseDamage damageSource)
```

Comment:
We have an interface for when a harpoon interacts with something. I feel that makes more sense to use that rather than health. Leaving the same comment on the other task